src/views/Tree.vue

```diff
@@ -187,6 +187,9 @@ fragment TaskProxyData on TaskProxy {
   isHeld
   isQueued
   isRunahead
+  isXtriggered
+  isRetry
+  isWallclock
```

Comment:
The other views need this too. Search for `isHeld` and add these fields where you find it.